### ISSUE TITLE
--var command line argument doesnt work

### ISSUE THREAD

**Issue by mhaber-tophat** [tophat/sanity-runner#50]
**Describe the bug**
when trying to pass a variable to the sanity-runner via --var flag, the variable doesnt get set in global.SANITY_VARIABLES. global.SANITY_VARIABLES ends up as an empty dict. (works if you set via config file)

**To Reproduce**

```
describe('Bug', async () => {
        test('test', async () => {
                await expect(global.SANITY_VARIABLES).toEqual({Test: 123})
        },5500)
})
```

```
sanity-runner-macos --var Test=123
```

```
 FAIL  Bug (0.746s)

• Bug test
Error: expect(received).toEqual(expected)

Expected value to equal:
  {"Test": 123}
Received:
  {}
```

**Expected behavior**
test should pass

**Comment by mhaber-tophat** [comment]
the issue is when passing in via flag, the collectVariables function returns a Promise , not the atually dictionary.

ex output of printing the vars diciontary when passing in via flags:

```
Promise { { APP_ENV: 'C' } }
``

**Comment by mhaber-tophat** [comment]
fixed with PR #51
